modules/ROOT/pages/feature-flagging.adoc

```diff
@@ -370,6 +380,276 @@ Suppressed errors are treated as underlying causes that can also be matched by O
 *Issue ID*
 
 * W-11529823
+<.^|`com.mulesoft.dw.xml_reader.honourMixedContentStructure`
+|When this property is set to true, DataWeave retains a mixed-content structure instead of grouping text with mixed content into a single text field.
+
+*Available Since*
+
+* 4.5.0
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* W-11071481
+<.^|`ENFORCE_ERROR_TYPES_VALIDATION` 
+|When enabled, error types validations are enforced, even for error handlers/components that are not being referenced.
+This feature isn't configurable by system property.
+
+*Available Since*
+
+* 4.5.0
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* W-10619787
+<.^|`disable.attribute.parameter.whitespace.trimming`
+|When enabled, Mule trims whitespaces from parameter values defined at the attribute level in the DSL.
+
+*Available Since*
+
+* 4.5.0
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* MULE-19803
+<.^|`disable.pojo.text.parameter.whitespace.trimming`
+|When enabled, Mule trims whitespaces from CDATA text parameter of pojos in the DSL.
+
+*Available Since*
+
+* 4.5.0
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* MULE-20048
+<.^|`enforce.expression.validation`
+|When enabled, expression validations are enforced for `targetValue`, not allowing a literal value.
+
+*Available Since*
+
+* 4.5.0
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* MULE-19987
+<.^|`enforce.dw.expression.validation`
+|When enabled, expression validations are enforced for all DataWeave expressions.
+
+*Available Since*
+
+* 4.5.0
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* MULE-19967
+<.^|`force.runtime.profiling.consumers.enablement`
+|When enabled, profiling consumers implemented by Mule are enabled by default.
+
+*Available Since*
+
+* 4.5.0
+
+* 4.4.0-202202
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* MULE-19967
+<.^|`mule.disable.registryBootstrapOptionalEntries`
+|When enabled, `optional` attribute in entries in a `registry-bootstrap.properties` are ignored.
+
+*Available Since*
+
+* 4.5.0
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* W-10736301
+<.^|`mule.disable.applyObjectProcessor`
+|When enabled, `org.mule.runtime.core.privileged.registry.ObjectProcessor` implementations aren't applied on objects registered into the `SimpleRegistry`.
+
+*Available Since*
+
+* 4.5.0
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* MULE-11737
+<.^|`mule.deployment.validateAppModelWithRegionClassloader`
+|When enabled, the application model is validated with the region classloader. When disabled, it is validated with the application classloader.
```

Comment:
To avoid the passive voice and a distant reference for the "it", I'd say:
When enabled, the region classloader validates the application model. When disabled, the application classloader validates the application model.